templates/careers.html

```diff
@@ -8,7 +8,7 @@
 
 <section class="p-strip--suru-topped is-shallow u-no-padding--bottom">
   <div class="u-fixed-width">
-    <h1>We’re Hiring!</h1>
+    <h1>We're Hiring!</h1>
```

Comment:
Why are we changing this? It seems like it previously used a correct typographical apostrophe.

Reply:
We only use straight apostrophes and I just noticed it as I worked